packages/fast-tooling-react/src/css-property-editor/property-editor-row.props.ts

```diff
@@ -0,0 +1,62 @@
+import { ManagedClasses } from "@microsoft/fast-jss-manager-react";
+
+export interface CSSPropertyEditorRowClassNameContract {
+    cssPropertyEditorRow?: string;
+    cssPropertyEditorRow_input?: string;
+    cssPropertyEditorRow_inputKey?: string;
+    cssPropertyEditorRow_inputValue?: string;
+}
+
+export interface CSSPropertyEditorRowUnhandledProps
+    extends React.HTMLAttributes<HTMLDivElement> {}
+
+export interface CSSPropertyEditorRowHandledProps
+    extends ManagedClasses<CSSPropertyEditorRowClassNameContract> {
+    /**
+     * The key associated with the value being edited
+     */
+    rowIndex: number;
+
+    /**
+     * The key associated with the value being edited
+     */
+    cssKey: string;
```

Comment:
if the value is just called `value` can we just call this `key`?

Reply:
We can't.  The "key" prop on the row used as the React Key for the row (see propertty-editor.tsx "renderRow").  Don't think React even passes that value down to the component (but in this case they aren't always the same).